addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SqlInjectionScanRule.java

```diff
@@ -1879,6 +1887,20 @@ private float responseBodyHeuristic(ComparableResponse one, ComparableResponse t
         return 0;
     }
 
+    private float locationHeaderHeuristic(ComparableResponse one, ComparableResponse two) {
+        if (one.getStatusCode() == two.getStatusCode()
+                && one.getStatusCode() >= 300
+                && one.getStatusCode() < 400) {
```

Comment:
I believe core has an isRedirect or similar, either in StatusCode or HttpResponseHeader (iirc)

Reply:
Done.